project3/styles.css

```diff
@@ -0,0 +1,373 @@
+    /* font-family: var(--font-sans); */
+    font-family: 'BenchNine', sans-serif;
+    font-size: 22px;
+    position: relative;
+}
+
+
+
+.box-2 {
+    background-color: #1e2019;
+    margin-top: -20px;
+    margin-bottom: 47px;
+    margin-left: 24px;
+    margin-right: 24px;
+    position: absolute;
+    height: auto;
+    width: 327px;
+
+
+}
+
+
+
+.realname {
+    width: 100%;
+}
+
+.alias {
+    width: 100%;
+}
+
+.input-box {
+    margin-left: 16px;
+}
+
+.agebox {
+    height: 68px;
+    width: 109px;
+    margin-left: 16px;
+}
+
+.powers-and-abilities {
+    margin-left: 18px;
+    text-align: left;
+}
+
+.describe-powers {
+    height: 220px;
+    width: 291px;
+}
+
+.test {
+    width: 100%;
+}
+
+.sr-only {
+    border: 0;
+    clip: rect(0 0 0 0);
+    overflow: hidden;
+    margin: -1px;
+    padding: 0;
+    position: relative;
+    height: 1px;
+    width: 1px;
+}
+
+
+.button-reset {
+
+    /* background: transparent; */
+    background-color: #fff;
+    border: 10px solid #71f79f;
+    border-width: 10px;
+    cursor: pointer;
+    font-family: Teko;
+    font-size: 32px;
+    font-weight: inherit;
+    padding: 0;
+    text-align: center;
+    width: 205px;
+    height: 87px;
+    display: block;
+    margin-right: auto;
+    margin-left: auto;
+    margin-bottom: 80px;
+    margin-top: 45px;
+    border-style: solid;
+}
+
+
+.button-reset:hover {
+    background-color: #71f79f;
+    /* border: 10px solid #71f79f; */
+    border: 10px solid #1e2019;
+    border-style: double;
+}
+
+.button-reset[disabled] {
+    cursor: default;
+}
+
+fieldset {
+    border: 0;
+    padding: 0;
+}
+
+input,
+textarea,
+select {
+    font-family: inherit;
+    font-size: inherit;
+    margin-left: 16px;
+    margin-right: 16px;
+
+}
+
+select {
+    cursor: pointer;
+    height: 60px;
+    width: 80px;
+}
+
+
+
+input[type="radio"],
+input[type="checkbox"] {
+    -moz-appearance: none;
+    -webkit-appearance: none;
+    appearance: none;
+    border: 3px solid #fff;
+    height: 12px;
+    width: 12px;
+}
+
+input[type="radio"] {
+    border-radius: 50%;
+}
+
+input[type="radio"]:focus,
+input[type="checkbox"]:focus {
+    border: 3px solid #71f79f;
+}
+
+input[type="radio"]:hover,
+input[type="checkbox"]:hover {
+    border: 3px solid #71f79f;
+}
+
+input[type="radio"]:checked,
+input[type="checkbox"]:checked {
+    background-color: #71f79f;
+    border: 3px solid #fff;
+
+    text-decoration-line: underline;
+    text-decoration-color: #71f79f;
+}
+
+input[type="real-name"]:invalid,
+input[type="email"]:not(:placeholder-shown):invalid,
+input[type="alias"]:not(:placeholder-shown):invalid {
+    background-color: #FFF;
+    width: 291px;
+    margin-left: 16px;
+    margin-right: 16px;
+    position: relative;
+    overflow: auto;
+}
+
+@media (min-width:375px) {
+    body {
+        background-color: #163346;
+    }
+}
+
+@media (min-width:520px) {
+    body {
+        background-color: #163346;
+    }
+}
```

Comment:
You do not need to repeat rules in your media query. If you set a value outside of the media query, it will continue for all sizes -- you only write rules for what _changes_ in media queries.